tests/rules/metavar_type_decorator_python.py

```diff
@@ -0,0 +1,3 @@
+@x.bar()
```

Comment:
would have been good to add a comment in the target file saying
it's normal there is no ruleid: here because we just care whether semgrep-core crash or not.